tests/components/unifi/test_switch.py

```diff
@@ -263,13 +265,13 @@ async def test_switches(hass):
     )
 
     assert len(controller.mock_requests) == 4
-    assert len(hass.states.async_entity_ids("switch")) == 3
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 3
 
     switch_1 = hass.states.get("switch.poe_client_1")
     assert switch_1 is not None
     assert switch_1.state == "on"
     assert switch_1.attributes["power"] == "2.56"
-    assert switch_1.attributes["switch"] == "00:00:00:00:01:01"
+    assert switch_1.attributes[SWITCH_DOMAIN] == "00:00:00:00:01:01"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)